src/main/java/de/ait/secondlife/services/OfferServiceImpl.java

```diff
@@ -91,13 +97,14 @@ public OfferResponseDto createOffer(OfferCreationDto dto) throws CredentialExcep
                     throw new WrongAuctionParameterException("winBid");
                 }
             }
-            newOffer = offerRepository.save(newOffer);
```

Comment:
the saving of the offer in DB must be before verify(newOffer) and draft(newOffer). 
please leave offerRepository.save() here, because now the code is broken and a new Offer cannot be created (error 400)

Reply:
fixed